src/main/resources/templates/c_plus_plus/readme

```diff
@@ -0,0 +1,15 @@
+1. [task][CMake runs correctly](TestConfigure)
+2. [task][Your code compiles](CompileSort)
+3. [task][Code is formatted with clang-format](clang-format(sort.cpp))
+
+
+# Sorting Algorithms
+[task][All algorithms sort correctly](TestCatch2(sort-test))
+1. [task][Selection Sort](sorting_algorithms/selection_sort,sorting_algorithms/all_elements_equal/selection_sort,sorting_algorithms/reverse-sorted_values/selection_sort,sorting_algorithms/single_values/selection_sort,sorting_algorithms/empty_input/selection_sort,sorting_algorithms/large_input/selection_sort)
```

Comment:
Can you add some text about each sorting algorithm. I don't know what "Explicit Heapsort" is supposed to mean and the template should also be a good example of an exercise

Reply:
This was accompanied by an exercise sheet with more details about the tasks and some visualizations. The difference between the two is the hand-written implementation using primitives (make_heap + pop_heap) vs. the reliance on higher-level standard-library functions (make_heap + sort_heap)

Reply:
Here are the descriptions, feel free to use them for the template @magaupp 
<detail>
<summary>Task descriptions</summary>
<img width="775" alt="Screenshot 2024-10-08 at 14 21 04" src="https://github.com/user-attachments/assets/a7164f7c-55f5-4f87-a54c-94c0422b42ad">
<img width="775" alt="Screenshot 2024-10-08 at 14 21 15" src="https://github.com/user-attachments/assets/0f739843-2232-4980-8cf5-c1ccf0efe553">
<img width="775" alt="Screenshot 2024-10-08 at 14 21 27" src="https://github.com/user-attachments/assets/324a0110-5c61-4c32-8f2c-fbcfe9b9d8de">
<img width="775" alt="Screenshot 2024-10-08 at 14 21 40" src="https://github.com/user-attachments/assets/c5457fe2-021b-464e-993c-6f17988e26a9">
<img width="775" alt="Screenshot 2024-10-08 at 14 22 17" src="https://github.com/user-attachments/assets/d0e49908-24e7-4b3a-ba10-c3aaf04f1ddf">
</detail>

Reply:
I can add the text descriptions. But it is currently not possible to provide images from the templates.
I'll add image support in a follow-up PR.